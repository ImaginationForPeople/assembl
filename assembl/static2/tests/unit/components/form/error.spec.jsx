import React from 'react';
import renderer from 'react-test-renderer';

import DummyForm from './dummyForm';
import Error from '../../../../js/app/components/form/error';

describe('Error component', () => {
  it('should render a field error', () => {
    const props = {
      name: 'foobar'
    };

    const component = renderer.create(
      <DummyForm>
        <Error {...props} />
      </DummyForm>
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});