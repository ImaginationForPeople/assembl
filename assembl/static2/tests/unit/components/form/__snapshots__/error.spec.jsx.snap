// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error component should render a field error 1`] = `
<form
  onSubmit={[Function]}
>
  <button
    disabled={true}
    type="submit"
  >
    Submit
  </button>
</form>
`;
